aksparse/client/aksparse/helpers.py

```diff
@@ -0,0 +1,71 @@
+import arkouda as ak
+
+def gen_ranges(starts, ends):
```

Comment:
A function similar to this exists in `arkouda/segarray.py`. Maybe we should look at using the code there and updating with any additional functionality we have here.

Reply:
This package in particular is a bit older so it could probably stand to be significantly updated (and probably some compatibility issues with newer arkouda versions).

Reply:
Added issue #32 to track